docs/deployment/connect-scm.md

```diff
@@ -63,52 +63,153 @@ You have successfully connected an org in Semgrep Cloud Platform with an organiz
 
 For users of GitLab cloud-hosted plans, a connection to GitLab is created automatically after [adding a Semgrep job to GitLab CI/CD](/deployment/add-semgrep-to-ci/). No other steps are needed.
 
-## Connect to on-premise GitHub or GitLab orgs
-
-### Applicable plans
-
-| GitHub | GitLab |
-| ------ | ------ |
-| GitHub Enterprise Server       | Any GitLab Self Managed plan        |
-
-### Table of required scopes for PATs
-
-Semgrep Cloud Platform requires personal access tokens (PATs) with assigned scopes. These scopes grant necessary permissions to the PAT and vary depending on the user's plan.
-
-| GitHub Enterprise Server          | GitLab Self-Managed        |
-|:---------------------------|:---------------------------|
-| <ul><li>`public_repo`</li> <li>`repo:status`</li> <li>`user:email`</li> <li>`write:discussion`</li></ul> | `api` |
-
-### Make the connection
+## Connect to on-premise orgs
+
+### GitHub Enterprise Server
+
+This section is applicable to users with **GitHub Enterprise Server**. We
+recommend connecting to your GitHub orgs using the Semgrep App instead of using
+a personal access token (PAT) whenever possible.
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs
+    defaultValue="app"
+    values={[
+    {label: 'Semgrep App (Recommended)', value: 'app'},
+    {label: 'PAT', value: 'pat'},
+    ]}
+>
+
+<TabItem value='app'>
+
+The Semgrep App for GitHub Enterprise (GHE) creates a connection between Semgrep
+and orgs in your GHE deployment. There are two primary installation
+steps:
+
+1. Install the Semgrep App for the first time using the GHE organization (org)
+   that "owns" the app.
+2. Install the app for additional GHE orgs.
+
+<!-->Using HTML headers instead of MD headers because I don't want them in the right-side nav<-->
```

Comment:
the docusaurus frontmatter option for header levels is what controls what appears in the nav:
https://docusaurus.io/docs/next/markdown-features/toc#table-of-contents-heading-level

For example:
```
# Display h2 to h5 headings
toc_min_heading_level: 2
toc_max_heading_level: 5
```